package.json

```diff
@@ -29,9 +33,10 @@
   "dependencies": {
     "bindings": "^1.2.0",
     "debug": "^2.2.0",
-    "nan": "^2.2.0"
+    "node-addon-api": "^1.6.2"
   },
   "devDependencies": {
     "mocha": "*"
-  }
+  },
+  "gypfile": true
```

Comment:
I'm not sure that this is required. `npm` adds it implicitly to the package metadata with the presence of a `binding.gyp` file.

Reply:
checking it out ...
Im sure you are correct.

Reply:
yes indeed so, it was in the first hello-world example for NAPI conversion but in no others from the workshop I looked at. I'll raise a PR on the workshop too.